scout/adapter/mongo/clinvar.py

```diff
@@ -274,6 +275,34 @@ def clinvar_objs(self, submission_id, key_id):
 
         return None
 
+    def rename_casedata_samples(self, submission_id, old_name, new_name):
+        """Rename all samples associated to a clinVar submission
+
+        Args:
+            submission_id(str): the _id key of a clinvar submission
+            old_name(str): old name of an individual in case data
+            new_name(str): new name of an individual in case data
+
+        Returns:
+            renamed_samples(int)
+        """
+        renamed_samples = 0
+        LOG.info(f"Renaming clinvar submission samples for submission id {submission_id}")
+
+        casedata_objs = self.clinvar_objs(submission_id, "case_data")
+
+        for obj in casedata_objs:
+            if obj.get("individual_id") == old_name:
```

Comment:
I don't perhaps feel strongly that this has to be a very tightly controlled feature, but as I'm reviewing.. This does run the risk of a user inadvertently "anchoring" multiple names. I e individuals named Alice and Bob from cases A and B. Now user edits "Bob" to "Alice". Next time either name changes, both will, in both cases.

Reply:
Fixed this by passing an additional param to the function (case):

![image](https://user-images.githubusercontent.com/28093618/84994408-ef608d80-b14a-11ea-8a29-eb583533b83e.png)

That results in an additional check on what to update in the adapter:
![image](https://user-images.githubusercontent.com/28093618/84994593-32bafc00-b14b-11ea-963c-ddc2ee0e8fdb.png)